spec/models/note_spec.rb

```diff
@@ -127,7 +127,7 @@
     end
     context 'when versions are turned off' do
       before do
-        Setting['advanced.versions'] = false
+        ENV['versions'] = 'false'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.